### ISSUE TITLE
feature request: support for the new SNS PublishBatch API

### ISSUE THREAD

**Issue by alexwlchan** [localstack/localstack#5057]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature description

AWS recently announced a new PublishBatch API for SNS, which allows you to send up to 10 messages in a single API request. It'd be good to have that API in LocalStack.

### 🧑‍💻 Implementation

There's already an SNS implementation in LocalStack, so I assume this involves adding a new API to that.

### Anything else?

AWS blog post announcing the issue: https://aws.amazon.com/blogs/compute/publishing-messages-in-batch-to-amazon-sns-topics/

API reference: https://docs.aws.amazon.com/sns/latest/api/API_PublishBatch.html

**Comment by rdlowrey** [comment]
+1 ... use localstack heavily for development and ran into this error message today when attempting to implement the new `PublishBatch` API:

```
InternalFailure: API action 'PublishBatch' for service 'sns' not yet implemented
```

**Comment by whummer** [comment]
Thanks for reporting @alexwlchan @rdlowrey @liviu-bot . This should be fixed in #5215 - can you please pull the `latest` Docker image and give it another try? Please let us know if the problem persists.. Thanks!

**Comment by whummer** [comment]
Thanks for the pointer @liviu-bot . Looks like we've had a small hickup in our CI pipeline. The build on `master` is now running again - you can follow the progress [here](https://app.circleci.com/pipelines/github/localstack/localstack/3486/workflows/d16ea55b-fbf6-4014-8da5-fce7513f7946). The `latest` image should become available in ~40mins. Apologies for the confusion!
